docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Create and explore a database using YCQL
+headerTitle: Create and explore a database using YCQL
+linkTitle: Create a database
+description: Create and explore a database using YCQL.
+headcontent:
+image: /images/section_icons/quick_start/explore_ycql.png
+menu:
+  latest:
+    parent: cloud-basics
+    name: Create a database
+    identifier: create-databases-2-ycql
+    weight: 60
+type: page
+isTocNested: false
+showAsideToc: true
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="../create-databases/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL
+    </a>
+  </li>
+
+ <li >
+    <a href="../create-databases-ycql/" class="nav-link active">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  
+</ul>
+
+Using the `ycqlsh` shell, you can interact with your YugabyteDB database using the YCQL API. The following exercise shows how you can use ycqlsh to create a keyspace, add a table, insert some data, and run a simple query.
+
+## Create a keyspace and add a table
+
+To create a keyspace and add a table, do the following:
+
+1. Connect to your cluster using `ycqlsh` using the [Client Shell](../connect-to-clusters#connect-via-client-shell) from your computer.
```

Comment:
Do these instructions apply to the cloud shell, as well? (If so, worth mentioning, as that's easier.)